boot/freeldr/freeldr/ntldr/arch/i386/winldr.c

```diff
@@ -15,12 +15,6 @@
 #include <debug.h>
 DBG_DEFAULT_CHANNEL(WINDOWS);
 
-// This is needed because headers define wrong one for ReactOS
-#undef KIP0PCRADDRESS
-#define KIP0PCRADDRESS                      0xffdff000
```

Comment:
So this definition has already been fixed in our headers?

Reply:
This is repeated in i386.h